src/Features/header/components/Sidebar/SidebarSlice.ts

```diff
@@ -0,0 +1,50 @@
+import { createApi } from '@reduxjs/toolkit/query/react';
+import type { BaseQueryFn } from '@reduxjs/toolkit/query/react';
+import axios from 'axios';
+import type { AxiosRequestConfig, AxiosError } from 'axios';
+
+const axiosBaseQuery =
```

Comment:
again cool! but you need to create `src/services/baseQuery.ts` file and put it there. Because we will have more hooks that will use `axiosBaseQuery` , and it will strange why the axios wrapper comes only from cities